### PR TITLE
Improved to accept null values

### PR DESCRIPTION
**Pull request by tempate**

### CODE REVIEW

File: commands/match.py [hunk 1]
```diff
@@ -52,8 +56,8 @@ def erf_inv(x):
         def phi_inv(p):
             return math.sqrt(2.0) * erf_inv(2.0 * p - 1.0)
 
-        mu_min = m_mu + phi_inv(0.025) * m_std_dev
-        mu_max = m_mu + phi_inv(0.975) * m_std_dev
+        mu_min = max(m_mu + phi_inv(0.025) * m_std_dev, np.nextafter(0,1))
+        mu_max = min(m_mu + phi_inv(0.975) * m_std_dev, np.nextafter(1,0))
```
Comment (Mk-Chan) [inline commands/match.py]: Can't you make do without numpy? I'd prefer not adding an extra dependency. Also formatting (There should be a space after the comma)

**Review by Mk-Chan — CHANGES_REQUESTED** [review]

File: commands/match.py (hunk 1 above)
Comment (tempate) [reply commands/match.py]: Done!

**Comment by Mk-Chan** [comment]
```
commands/match.py", line 25
    if wins < 0 or losses < 0 or draws < 0:
                                          ^
```
That's cuz you used tabs instead of spaces. That's ok, I'll fix it

**Comment by Mk-Chan** [comment]
Thanks

**Comment by Mk-Chan** [comment]
This commit doesn't work the error bar is screwed. Force pushed out.

**Pull request merged**